bluepyopt/ephys/protocols.py

```diff
@@ -193,26 +206,30 @@ def _reduce_method(meth):
             import copyreg
             import types
             copyreg.pickle(types.MethodType, _reduce_method)
+            import pebble
+            from concurrent.futures import TimeoutError
 
-            import multiprocessing
+            if timeout is not None:
+                timeout = max(timeout, 0)
```

Comment:
Wouldn't it be better to raise a ValueError when a negative timeout is given ? This implicit assumption might not be clear